hack/upgrade-test.sh

```diff
@@ -211,22 +211,27 @@ ${CMD} wait pod $CATALOG_OPERATOR_POD --for condition=Ready -n openshift-operato
 #  currentCSV: kubevirt-hyperconverged-operator.v100.0.0
 #  installedCSV: kubevirt-hyperconverged-operator.v100.0.0
 echo "--"
-echo "-- Upgrade Step 5/6: verify the subscription's currentCSV and installedCSV have moved to the new version"
+echo "-- Upgrade Step 5/7: verify the subscription's currentCSV and installedCSV have moved to the new version"
 echo "--"
 
-
 sleep 10
 HCO_OPERATOR_POD=`${CMD} get pods -n kubevirt-hyperconverged | grep hco-operator | head -1 | awk '{ print $1 }'`
 ${CMD} wait pod $HCO_OPERATOR_POD --for condition=Ready -n kubevirt-hyperconverged --timeout="600s"
 ./hack/retry.sh 30 60 "${CMD} get subscriptions -n kubevirt-hyperconverged -o yaml | grep currentCSV | grep v100.0.0"
 ./hack/retry.sh 2 30 "${CMD} get subscriptions -n kubevirt-hyperconverged -o yaml | grep installedCSV | grep v100.0.0"
 
 echo "--"
-echo "-- Upgrade Step 6/6: verify the hyperconverged-cluster deployment is using the new image"
+echo "-- Upgrade Step 6/7: verify the hyperconverged-cluster deployment is using the new image"
 echo "--"
 
 ./hack/retry.sh 6 30 "${CMD} get deployments -n kubevirt-hyperconverged -o yaml | grep image | grep hyperconverged-cluster | grep ${REGISTRY_IMAGE_URL_PREFIX}"
 
 echo "----- Images after upgrade"
 ${CMD} get deployments -n kubevirt-hyperconverged -o yaml | grep image | grep -v imagePullPolicy
 ${CMD} get pod $HCO_CATALOGSOURCE_POD -n ${HCO_CATALOG_NAMESPACE} -o yaml | grep image | grep -v imagePullPolicy
+
+echo "-- Upgrade Step 7/7: wait that cluster will become operational"
+make check-state
```

Comment:
Can we also perform a check on statuses before the upgrade is started.

Reply:
done